### ISSUE TITLE
RCS1228 triggered for primary constructor parameters

### ISSUE THREAD

**Issue by gtbuchanan** [dotnet/roslynator#1247]
**Product and Version Used**: 
Roslynator 4.6.2

**Steps to Reproduce**:
Create a `class` with a primary constructor and document the parameters as you would with a `record` primary constructor.

**Actual Behavior**:
```csharp
/// <summary>
/// This is a class.
/// </summary>
/// <param name="value">This is a value.</param>
internal sealed class MyClass(string value)
{
    public string Value { get; } = value;
}
```
![image](https://github.com/dotnet/roslynator/assets/715687/b1c58014-ade1-4389-999f-22540489763f)

**Expected Behavior**:
RCS1228 should not be triggered.
